.github/workflows/rust.yml

```diff
@@ -33,14 +33,25 @@ jobs:
           default: true
           override: true
           profile: minimal
-          components: rustfmt, clippy
+          components: rustfmt, clippy, llvm-tools-preview
 
       - name: Cache Dependencies
         uses: Swatinem/rust-cache@v1
-        
+
+      - name: Download grcov
+        run: |
+          mkdir -p "${HOME}/.local/bin"
+          curl -sL https://github.com/mozilla/grcov/releases/download/v0.8.11/grcov-x86_64-unknown-linux-gnu.tar.bz2 | tar jxf - -C "${HOME}/.local/bin"
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
       - name: Tests
         run: ./scripts/tests/${{ matrix.scripts }}.sh
 
+      - name: Upload to codecov.io
+        uses: codecov/codecov-action@v3
+        with:
+          files: /misc/tests.lcov
```

Comment:
Might want to use */tmp/tests.lcov* instead to avoid potential permission issues.

Reply:
addressed in https://github.com/zeitgeistpm/zeitgeist/pull/759/commits/eb06f3518ad20aecf7150f32c76b6bb2b8d17435